<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Poizvedbe</string>
     </attribute>
     <widget class="QLabel" name="status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QComboBox" name="izbira">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Knjige</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Avtorji</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Clani</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zalozbe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Knjiznicarji</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pozeni">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pozeni</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaj isces</string>
      </property>
     </widget>
     <widget class="QTableView" name="izpis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>1001</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Izbrisi</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idIzbris">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>0</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Izbris po ID</string>
      </property>
     </widget>
     <widget class="QComboBox" name="izbiraAtribut">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Po cem isces(prazno-za vse zapise)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="atributValue">
      <property name="geometry">
       <rect>
        <x>272</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>0</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Razvrsti po</string>
      </property>
     </widget>
     <widget class="QComboBox" name="izbiraSort">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Dodajanje</string>
     </attribute>
     <widget class="QRadioButton" name="radioKnjiga">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Knjiga</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioClan">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Clan</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioAvtor">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Avtor</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioZalozba">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Zalozba</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inEdit1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inEdit2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inEdit3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inEdit4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inEdit5">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inEdit6">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="inButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Vnos</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>2070</x>
        <y>260</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="inCombo">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Prosta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>V citalnici</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="testlabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Izposoja</string>
     </attribute>
     <widget class="QLineEdit" name="inKnjiga">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inKnjiznicar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inClan">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="izposoja">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Izposoja</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vrnitev">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Vrnitev</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
